packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -322,8 +324,12 @@ export default decorate([
   provideState({
     initialState: () => ({
       filter: undefined,
+      page: 1,
     }),
     effects: {
+      onPageChange(_, page) {
+        this.state.page = page
+      },
```

Comment:
```suggestion
      onPageChange: (_, page) => ({ page }),
```

Reply:
It's an old syntax which @julien-f doesn't recommend to use it.

Reply:
This is Freactal's syntax, which is currently supported in Reaclette but might go away in the future.